ansible/deploy

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```

Comment:
I personally find bash scripts to be pretty awful for long-term usage, especially when they're managed by multiple folks (everyone seems to always have their own style). This is pretty small though, so would something like Python perhaps work for the script?